code/modules/client/preference/preferences.dm

```diff
@@ -198,6 +198,8 @@ GLOBAL_LIST_INIT(special_role_times, list( //minimum age (in days) for accounts
 			dat += "<b>Gender:</b> <a href='?_src_=prefs;preference=gender'>[active_character.gender == MALE ? "Male" : (active_character.gender == FEMALE ? "Female" : "Genderless")]</a>"
 			dat += "<br>"
 			dat += "<b>Age:</b> <a href='?_src_=prefs;preference=age;task=input'>[active_character.age]</a><br>"
+			if(active_character.age < S.min_age || active_character.age > S.max_age)
+				active_character.age = rand(S.min_age, S.max_age)
```

Comment:
Could you please use a clamp here so that a 85 year old vulp isnt set to 17, but rounded down to their max age instead?

Reply:
Like this?

Reply:
Close I think, it should probably be changed in the back end (like /Topic() and href stuff) and not in the front end like it is currently (dat += ...)

Reply:
Any idea where? I did that here originally to prevent abuses like selecting 300 years on drasks and swapping to human so they can have it without clicking on "age".